### PR TITLE
Project 4: Apple Website Clone

### PR DESCRIPTION
**Pull request by praz99**
### In this project I learned many new things in HTML and CSS including:
* Working with Gradients.
  * Linear Gradients.
  * Radial  Gradients.
* Working with Backgrounds.
* Various properties of backgrounds in CSS.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @praz99 
### Good job done so far :clap: However, you have a few issues to solve before your application can be approved.
- Configure linting properly to help with the errors produced, check this [guide to help](https://github.com/microverseinc/linters-config/tree/master/css)
- Add logo to your project
logo
### Make these changes and resubmit for another code review. Good luck.
- Contact me on slack @iamraymond for clarity
- Happy coding!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                    <img src="img/your_verse_subhead_small.png" alt="Everyone has something to share">
+                </h2>
+                <h3>
+                    <img src="img/cta_your_verse_27pt.png" alt="something here">
+                </h3>
+            </div>
+        </header>
+    </main>
+    <aside>
+        <img class="grid-item iphone5s" src="img/promo_iphone5s.jpg" alt="iphone 5S">
```
Comment (uryelah) [inline index.html]: - [ ] Each image in the aside should be wrapped inside an `<a>` tag.
Screenshot_2020-03-23 dist html_css building_with_backgrounds_and_gradients md · master · Microverse guides tse code_review

**Review by uryelah — CHANGES_REQUESTED** [review]
Hello **Thapa Thapa**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, great job with the previous review fixes!🎇**
Your clone is looking great. You are pretty much just missing one requirement for this project.

---------    

### Required changes 🛠 
---------
**Check the required changes in the comments below** 🔽

**Comment by uryelah** [comment]
**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by uryelah** [comment]
Good job adding the links.

Your project is looking great, as an [OPTIONAL] improvement I'd ask you to also implement transitions on your nav hover effect. It makes for a much richer user experience, [you can read more about transitions here.](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions)

------------

## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

-------

**Keep being an awesome dev, good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by praz99** [comment]
Thank you so much for approving the project. And I have implemented the
transition too.

Regards

Thapa

**Pull request merged**